frameworks/Ruby/rails/app/controllers/hello_world_controller.rb

```diff
@@ -24,9 +24,9 @@ def query
   end
   
   def fortune
-    @fortunes = Fortune.all
+    @fortunes = Fortune.order(:message)
     @fortunes << Fortune.new(:id => 0, :message => "Additional fortune added at request time.")
-    @fortunes = @fortunes.sort_by { |x| x.message }
+    @fortunes.sort_by!(&:message)
```

Comment:
I might be reading the code incorrectly but isn't this sorting twice now? Once in the database and then again in the Ruby?. I also have a feeling that I benchmarked this and it was faster to sort in code, rather then to do it in the database.